doc/man1/openssl-cms.pod.in

```diff
@@ -452,9 +452,11 @@ used multiple times if more than one signer is required.
 
 =item B<-certfile> I<file>
 
-Allows additional certificates to be specified. When signing these will
-be included with the message. When verifying these will be searched for
-the signers certificates.
+Provide additional certificates.
+When signing, these will be included
+in the message to help chain building and to provide extra signer certificates.
+When verifying, these will be searched first for signer certificates.
```

Comment:
I had to adapt the first suggested line because when signing the certs may also serve as fallback signer certs.

Nice extra text/notes in the last three suggested lines.